src/.editorconfig

```diff
@@ -0,0 +1,123 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+###############################
+# Core EditorConfig Options   #
+###############################
+# All files
+[*]
+indent_style = space
+# Code files
+[*.{cs,csx,vb,vbx}]
+indent_size = 4
+insert_final_newline = true
+charset = utf-8-bom
```

Comment:
I would recommend simply `utf-8`. I know a lot of tools (and windows) prefer `utf-8-bom` but the `BOM` is irrelevant for `utf-8`. see also this http://utf8everywhere.org/#faq.boms and this https://en.wikipedia.org/wiki/UTF-8#Byte_order_mark.